src/scribe_data/cli/get.py

```diff
@@ -177,25 +183,30 @@ def prompt_user_download_all():
     # MARK: Translations
 
     elif data_type == "translations":
+        # If no language specified, use "all".
         if language is None:
             language = "all"
         parse_wd_lexeme_dump(
             language=language,
             wikidata_dump_type=["translations"],
             type_output_dir=output_dir,
             wikidata_dump_path=wikidata_dump,
+            overwrite_all=overwrite,
         )
         return
 
     # MARK: Form Dump
 
-    elif wikidata_dump:
+    elif wikidata_dump is not None:
+        if not wikidata_dump:
```

Comment:
Just wondering for myself -
but this is simply just a check for if `wikidata_dump` is an empty string `""`? Did I get that right?